src/styles/global.css

```
@@ -30,10 +30,26 @@ button { cursor: pointer; font-family: inherit; }
+  background: var(--bg);
+}
+::-webkit-scrollbar-thumb {
+  background: var(--border);
+  border-radius: 3px;
+}
+::selection {
+  background: rgba(252, 211, 77, 0.9);
+  color: #000000;
+}
 
+::-moz-selection {
```

Stylelint (17.12.0)

[error] Vendor-prefixed selector "::-moz-selection" (selector-no-vendor-prefix)

(selector-no-vendor-prefix)